data_safe_haven/config/sre_config.py

```diff
@@ -98,7 +98,10 @@ def template(cls: type[Self], tier: int | None = None) -> SREConfig:
                     allow_copy=remote_desktop_allow_copy,
                     allow_paste=remote_desktop_allow_paste,
                 ),
-                research_user_ip_addresses=["List of IP addresses belonging to users"],
+                research_user_ip_addresses=[
+                    "List of IP addresses belonging to users",
+                    "You can also use the tag 'Internet' instead of a list",
```

Comment:
```suggestion
                    "You can also use an Azure ServiceTag (such as 'Internet') instead of a list",
```

Reply:
Also - should this be a `list[str | AzureServiceTag]` instead of `list[str] | AzureServiceTag`?

Reply:
That isn't right, only the Internet tag can be used.

In that case, it seemed sensible for the type to be tag or list of IPs rather than list of IPs and tags.